lib/libzutil/os/linux/zutil_import_os.c

```diff
@@ -650,7 +651,11 @@ zpool_label_disk_wait(const char *path, int timeout_ms)
 		udev_device_unref(dev);
 		(void) usleep(sleep_ms * MILLISEC);
 
-	} while (NSEC2MSEC(gethrtime() - start) < timeout_ms);
+	} while ((c = (NSEC2MSEC(gethrtime() - start) < timeout_ms)));
+
+	if (c == B_FALSE)
+		fprintf(stderr, "error: %s",
+		    "ZPOOL_IMPORT_UDEV_TIMEOUT_MS exceeded\n");
```

Comment:
We should avoid printing to stderr/stdout from within the library.  Let's instead return `ETIMEDOUT` and update all the callers which are already logging the error as appropriate on failure.